crates/ruff_db/src/diagnostic/render/snapshots/ruff_db__diagnostic__render__json__tests__notebook_output.snap

```diff
@@ -10,7 +10,7 @@ expression: env.render_diagnostics(&diagnostics)
       "column": 10,
       "row": 2
     },
-    "filename": "notebook.ipynb",
+    "filename": "/notebook.ipynb",
```

Comment:
Were those paths (and in other output formatters) relative or absolute before you migrated the renderers to the new diagnostic output? Some folks might depend on paths being relative or absolute.

Reply:
I guess I'm slightly confused why the snapshots change...